docs/howto-navigate-history.md

```diff
@@ -0,0 +1,21 @@
+# How to explore a history of versions
```

Comment:
Why the file is named `howto-...` and not `how-to-...` ? (it may be that my English is not good enough)

Reply:
As explained in https://github.com/ambanum/OpenTermsArchive/pull/971#discussion_r1040359603, the idea was to have categories as prefixes, thus making them stand in a single “namespace” of sorts. I agree this could be improved.